docs/core/tools/csproj.md

```diff
@@ -19,13 +19,26 @@ see [the MSBuild project file](https://docs.microsoft.com/visualstudio/msbuild/m
 
 ## Implicit package references
 Metapackages are now implicitly referenced based on the target framework specified in the `<TargetFramework>` or `<TargetFrameworks>` property of your project file. 
```

Comment:
Sorry, I probably should have referred to the sentence and the header. I meant to add that sentence here. This sentence is repeated now in the next section where you added the line.

> Metapackages are implicitly referenced based on the target framework specified in the `<TargetFramework>` or `<TargetFrameworks>` property of your project file. `<TargetFrameworks>` is ignored if `<TargetFramework>` is specified, independent of order.